portfolio/src/test/java/com/google/sps/TaskPriorityTest.java

```diff
@@ -0,0 +1,70 @@
+package com.google.sps;
+
+import com.google.sps.data.TaskPriority;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class TaskPriorityTest {
+  private static final int MAX_ACCEPTABLE_VALUE = 5;
+  private static final int MIN_ACCEPTABLE_VALUE = 1;
```

Comment:
As I mentioned earlier, these can be public constants in TaskPriority.

Reply:
Done.